README.md

```diff
@@ -106,6 +106,133 @@ aio commerce init
 1. Update Storefront to use build.mjs to modify pdp schema
 1. Use this data somehow, ie via slot context, and remove runtime fetch
 
+## Adobe Commerce Integration Starter Kit (Phase 4)
+
+The purpose of this exercise is to demonstrate sending events from Commerce to Adobe Developer App Builder. You will use a simplified version of the Adobe Commerce Integration Starter Kit to configure Commerce and an App Builder project, allowing for the sending of events about saved orders.
+
+### 1. Configure the Starter Kit
+
+1. Login to the Adobe Developer Console at https://developer.adobe.com/console/ and select the **Commerce Extensibility Lab** organization. 
+    
+    Developer Console is the gateway to access all the services and tooling available as part of the Adobe Developer Ecosystem, including App Builder. 
+
+    Use the username format:
+    
+    **pd+sj+\<SEAT_NUMBER\>@adobeevents.com**
+
+1. Select the project assigned to your seat: 
+
+    **Cloud Service <SEAT_NUMBER>**
+
+    *Tip: Use search to find your project*
+
+    Select the **Stage** workspace.
+
+    Download the workspace JSON configuration file by clicking on the **Download all** button on the upper right corner of the page. Save it as `workspace.json` at `extensibility-lab/scripts/onboarding/config/workspace.json`
+
+1.  Copy the `env.dist` file in `extensibility-lab` as `.env` 
+    ```
+    cp env.dist .env
+    ```
+
+    Then configure the `.env` file with all the information required:
+
+    - To find the values for the OAuth config variables, click **OAuth Server-to-Server** in the Developer Console Workspace overview page for the Stage workspace.
+
+        ![Alt text](docs/oauth-credential.png "OAuth Server-to-Server Credential")
+
+    - Set the REST endpoint for your ACCS instance as the value for the `COMMERCE_BASE_URL`. Note that the value should end with `/`.
```

Comment:
If the only change in the URLs used by different attendees is going to be the tenant_id, maybe we could include the common parts of the URL so they can just copy & paste it.

Reply:
Added an example url